src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -35,6 +35,11 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
         /// </summary>
         internal BinderFlags TopLevelBinderFlags { get; private set; }
 
+        /// <summary>
+        /// Whether Nullable Reference Types feature is enabled globally.
+        /// </summary>
+        public bool? Nullable { get; private set; }
```

Comment:
>bool? [](start = 15, length = 5)

Why use a nullable bool that defaults to `null`, rather than a bool with default `false`? (like `AllowUnsafe` above)

Reply:
Seems a PROTOTYPE comment then.

---
In reply to: [224930060](https://github.com/dotnet/roslyn/pull/30479#discussion_r224930060) [](ancestors = 224930060,224928022)

Reply:
> Seems a PROTOTYPE comment then.

I'll open an issue for that

---
In reply to: [224934637](https://github.com/dotnet/roslyn/pull/30479#discussion_r224934637) [](ancestors = 224934637,224930060,224928022)